metasyn/provider.py

```diff
@@ -61,9 +61,13 @@
     from metasyn.config import VarConfig, VarConfigAccess
 
 class BaseDistributionProvider(ABC):
-    """Class that encapsulates a set of distributions.
+    """Base class for all distribution providers.
 
-    It has a property {var_type}_distributions for every var_type.
+    A distribution provider is a class that provides a set of distributions
+    that can be used by metasyn to generate synthetic data.
+    This class acts as a base class for creating specific distribution
+    providers. It encapsulates a set of distributions and has a property
+    {var_type}_distributions for every var_type.
```

Comment:
Not anymore! Currently they are all in one list. Maybe some documentation on the legacy distributions would be nice as well.

Reply:
Changed it, is this better?